teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -152,9 +198,114 @@ With regards to this checklist, reviewers should do the following:
 
 #### Stage 4 - For Reviewers Only
 
-* [ ] Assigned modules' folders properly structured within topics
-* [ ] Gem Amounts 
-  * Labs: calculated using John's Code 
-  * Activities: Predetermined amounts
-* [ ] Have curriculum proofread by writing team \(currently Victoria Xu @vkxu657\)
+* [ ] **Bit Learning Management System \(LMS\)**
+
+  We have a LMS built to help us teach students step by step how to tackle complex and technical problems. In addition, we are open source so we are looking forward to finishing up LMS and creating small enough issues to let anyone contribute to our project.
+
+For the LMS to work, each module has a folder structure to follow.
+
+Walk through example:
```

Comment:
this example section's content is great but could use more styling - take advantage of markdown!!